dist/src/main/java/io/camunda/application/StandaloneSchemaManager.java

```diff
@@ -91,6 +91,7 @@ public static void main(final String[] args) {
 
     final ExporterConfiguration exporterConfig = new ExporterConfiguration();
     exporterConfig.setConnect(connectConfiguration);
+    exporterConfig.getIndex().setPrefix(connectConfiguration.getIndexPrefix());
```

Comment:
Can we separate this into a different PR? I feel like this is completely something different to the purpose of this PR?